deb/openmediavault/workbench/src/app/core/services/dashboard-widget-config.service.ts

```diff
@@ -34,10 +37,16 @@ export class DashboardWidgetConfigService {
 
   private configsSource = new BehaviorSubject<DashboardWidgetConfig[]>([]);
 
+  private get platformDeviceType(): string {
+    return this.platform.ANDROID || this.platform.IOS ? 'mobile' : 'desktop';
```

Comment:
This is using CDK See here https://www.decodedfrontend.io/detect-browser-and-os-with-angular-cdk-platform-module/:
It's somewhat limited, but I am using the same device names as from here:
https://www.npmjs.com/package/ngx-device-detector?activeTab=readme

So if it's felt it's too restrictive, the npm version can always the implemented without breaking too much (Hopefully)